src/main/java/ladder/domain/Ladder.java

```diff
@@ -23,4 +23,12 @@ public static Ladder of(int height, Participants participants, LineDrawStrategy
     public List<Line> getLines() {
         return lines;
     }
+
+    public int move(int startIndex) {
+        int position = startIndex;
+        for (Line line : lines) {
+            position = line.move(position);
+        }
+        return position;
```

Comment:
```suggestion
        return lines.stream()
                .reduce(startIndex, (position, line) -> line.move(position), (a, b) -> a);
    }
```

reduce api 를 활용할 수 있을것 같아요 👍